src/OrchardCore.Build/Dependencies.props

```diff
@@ -61,11 +61,11 @@
     <PackageManagement Include="xunit" Version="2.6.1" />
     <PackageManagement Include="xunit.analyzers" Version="1.5.0" />
     <PackageManagement Include="xunit.runner.visualstudio" Version="2.5.3" />
-    <PackageManagement Include="YesSql" Version="3.5.0" />
-    <PackageManagement Include="YesSql.Abstractions" Version="3.5.0" />
-    <PackageManagement Include="YesSql.Core" Version="3.5.0" />
-    <PackageManagement Include="YesSql.Filters.Abstractions" Version="3.5.0" />
-    <PackageManagement Include="YesSql.Filters.Query" Version="3.5.0" />
+    <PackageManagement Include="YesSql" Version="4.0.0-beta-0001" />
```

Comment:
Shouldn't the "Found version-specific or distribution-specific runtime identifier(s): alpine-arm, alpine-arm64, alpine-x64. Affected libraries: SQLitePCLRaw.lib.e_sqlite3. In .NET 8.0 and higher, assets for version-specific and distribution-specific runtime identifiers will not be found by default." build warnings be gone here too? With 3.5.0 they are.

Reply:
The updates are not part of 0001.  I'll bump it up to 2 so we can remove the warning

Reply:
I release and updated 0002. The warning should go away